### DISCUSSION TITLE
Post your analysis from week 2 to your portfolio by Friday; complete your reviews sometime the following week

### DISCUSSION THREAD

**Discussion by eculler** [earthlab-education/Earth-Analytics-2023-01-Intro#250] [Announcements]
* Instructions are [in the wiki](https://github.com/earthlab-education/Earth-Analytics-2023-01-Intro/wiki/How-to...#post-your-analysis-to-your-portfolio)
* Normally, this part will be due with the rest of your assignment, but this first week we're going to break it up. Please try to get yours posted by Friday; you should have some time in the discussion section to do so.
* Once you have posted, I will assign you two reviewers. When you get the review assignment, please [leave a review (instructions in the wiki](https://github.com/earthlab-education/Earth-Analytics-2023-01-Intro/wiki/How-to...#review-a-portfolio-submission)

**Comment by ellenalamont17** [comment]
How do we determine which submissions we are assigned to review? If we search by assignee in the PR tab, does that bring up the assigned reviews or only our specific submissions?

**Reply by eculler** [reply]
Great question - try the following search:

`is:open is:pr user-review-requested:@me `
